conf/services/plastic-packaging-tax-account.yml

```diff
@@ -0,0 +1,21 @@
+serviceName: Plastic Packaging Tax
+serviceDescription: |
+  You can use this service to manage your Plastic Packaging Tax account, submit a return and make a payment
+serviceDomain: www.gov.uk
+serviceUrl: /plastic-packaging-tax
+contactFrontendServiceId: plastic-packaging-tax
+complianceStatus: partial
+accessibilityProblems:
+  - Some people may find parts of this service difficult to use when the Welsh language is selected. This is because not all of the content in the service has Welsh translations. This means a screenreader will have problems as it will try to pronounce the English as Welsh.
+milestones:
+  - description: |
+      When the Welsh language has been selected not all of the content on the page is translated or it is missing the required translations in the configuration file. This doesn’t meet WCAG 2.1 success criterion 3.1.2 Language of Parts. This will be fixed by 31 August 2022 by ensuring all content is translated into Welsh and has an entry in the Welsh configuration file.
+    date: 2022-08-31
```

Comment:
this bit renders "...This will be fixed by 31 August 2022 by ensuring all content is translated into Welsh and has an entry in the Welsh configuration file. **This will be fixed by 31 August 2022.**"
perhaps rephrase?